integration/kafka/src/main/java/alluxio/kafka/connect/AlluxioSinkConnectorConfig.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the “License”). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.kafka.connect;
+
+import alluxio.kafka.connect.format.JsonFormat;
+
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.config.ConfigDef.Type;
+
+import java.util.Map;
+
+/**
+ * This class is responsible for reading and storing allxuio connector config parameters.
+ */
+public class AlluxioSinkConnectorConfig extends AbstractConfig {
+
+  public static final String ALLUXIO_URL = "alluxio.url";
+  public static final String TOPICS_DIR = "topics.dir";
+  public static final String ROTATION_RECORD_NUM = "rotation.num";
+  public static final String ROTATION_TIME_INTERVAL_MS = "rotation.time.interval.ms";
+  public static final String ALLUXIO_FORMAT = "alluxio.format";
+  public static final int OFFSET_LENGTH = 10;
+  public static final String ALLUXIO_CONNECTOR_VERSION = "1.1.0";
+  public static final String RETRY_TIME_INTERVAL_MS = "retry.time.interval.ms";
+  private static final String ALLUXIO_URL_DOC =
+      "alluxio url address, eg. alluxio://localhost:19998";
+  private static final String TOPICS_DIR_DOC = "The top level directory name in alluxio";
+  private static final String ROTATION_RECORD_NUM_DOC =
+      "The record number of alluxio file rotation";
+  private static final String ROTATION_TIME_INTERVAL_MS_DOC =
+      "The time interval of alluxio file rotation";
+  private static final String ALLUXIO_FORMAT_DOC = "The converter format from kafka to alluxio";
+  private static final String RETRY_TIME_INTERVAL_MS_DOC =
+      "The retry interval in case of transient exception";
+  private static ConfigDef sConfig = new ConfigDef()
+      .define(ALLUXIO_URL, Type.STRING, "alluxio://localhost:19998", ConfigDef.Importance.HIGH,
```

Comment:
yes, when the connector runs, it will read configuration file. The hostname and port in the above code is default value when the configuration file can not be found.